src/renderer/components/virtual-table/index.tsx

```diff
@@ -179,7 +183,7 @@ const tableColumns: { [key: string]: ColDef } = {
             GenericTableHeader(params, { position: 'center' }),
         headerName: i18n.t('table.column.dateAdded'),
         suppressSizeToFit: true,
-        valueFormatter: (params: ValueFormatterParams) => formatDateAbsolute(params.value),
+        valueFormatter: (params: ValueFormatterParams) => formatDateAbsoluteUTC(params.value),
```

Comment:
This should be absolute, not absolute UTC.

Reply:
I thought `dateAdded` only specified a date, with no time information? In that case shouldn't it be processed as UTC?

Reply:
No. `dateAdded` has time zone (at least for Navidrome)

Reply:
Switched it out with the non-UTC version